### ISSUE TITLE
ULs not indenting the second line.

### ISSUE THREAD

**Issue by sstrubberg** [carbon-design-system/gatsby-theme-carbon#233]
Running gatsby-theme 1.4.0. Indents are wonky.

![image](https://user-images.githubusercontent.com/15822070/60998120-ee5d9400-a31d-11e9-9979-da4e4079ac44.png)

I ended up making a `bullets` class with the following css...

```

.bullets {
  ul {
    width: 100%;
  }

  li {
    margin-bottom: 1.5rem;
    list-style-position: inside;
    text-indent: -0.5rem;
    padding-left: 1rem;
  }

  li:before {
    margin-right: 0;
  }
}


```

Here are my results..

![image](https://user-images.githubusercontent.com/15822070/60998319-4d230d80-a31e-11e9-9ba0-2cb3cc9b4104.png)

**Comment by alisonjoseph** [comment]
We've updated to use the Carbon classnames and styles, however, I'm still seeing this issue. 

Weird, because it seems fine here https://codepen.io/alisonjoseph/pen/vqPjyd

**Comment by abbeyhrt** [comment]
This is weird! Since I added the new class names, i'll try to figure out and fix what's going on!

**Comment by vpicone** [comment]
Fixed with #247
